_posts/2023-11-02-election-competitiveness-in-charlotte-north-carolina/election-competitiveness-in-charlotte-north-carolina.Rmd

```diff
@@ -0,0 +1,76 @@
+
+This follows the results from the actual 2015 and 2022 elections in that the 2022 election was much less competitive with a far greater vote difference between Democrats and Republicans. Simulating the 2015 election data on the currently enacted map still produced results that were much more competitive in 2015 compared to 2022. Furthermore, the current level of competition in Charlotte elections seems to be accurate to the actual partisan breakdown, as the simulated plans for the 2022 elections were roughly balanced with the currently enacted plan, with four more competitive districts and three less competitive districts. As a result, these plots show that the decrease in competitiveness in Charlotte from 2015 to 2022 is likely due to demographic and political shifts in those years rather than the enacted plan reducing competitiveness.
+
+## Methods and Materials
+
+#### Data
+
+Election data came from the 2015 Mayoral election and the 2022 Mayoral and city council at-large elections. The North Carolina State Board of Elections organizes the election data by precinct, after which I kept only the candidates who were Democrats or Republicans in order to more clearly analyze the partisan competitiveness. One precinct with no voting data was removed, as well.
+
+The Charlotte council district and precinct maps came from the City of Charlotte Open Data Portal. The precinct map of Charlotte was merged with the precinct election data, which was then filtered down to the council district boundaries.
+
+Sources are linked below.
+
+-   [Election Data](https://er.ncsbe.gov/index.html?election_dt=07/26/2022&county_id=60&office=ALL&contest=0)
+-   [City of Charlotte GIS Map -- Council Districts](https://data.charlottenc.gov/datasets/charlotte::council-districts/explore?location=35.216425%2C-80.785856%2C10.00&style=District)
+-   [City of Charlotte GIS Map -- Precincts](https://www.nconemap.gov/datasets/voting-precincts/explore?filters=eyJjb3VudHlfaWQiOls2MCw2MF19&location=35.248115%2C-80.674966%2C10.43)
+
+### Methods
+
+I sampled 30,000 redistricting plans for North Carolina, across two independent runs of the SMC algorithm, then thinned the sample down to 5,000 plans.
+
+For constraints, I used the high school attendance zones in Charlotte as the neighborhood map. We use neighborhood maps when producing redistricting simulations as many cities attempt to preserve communities of interest when drawing new district maps, in order to avoid splitting a single neighborhood into multiple city council districts. Other potential neighborhood zones, like the Neighborhood Statistical Areas, historical districts, and Zip Codes were tested, but did not provide any benefit as a constraint because the neighborhood areas were too small or lacked coverage over the entire city. The neighborhood plot of school zones is shown below.
```

Comment:
Maybe also add some justification for why high school attendance zones make sense qualitatively as neighborhoods?